deployment/flexible_api_server.py

```
@@ -0,0 +1,560 @@
+                "suggestion": "Try again in a few seconds",
+                "text": text,
+                "deployment_type": "serverless"
+            }
+        except requests.exceptions.RequestException as e:
+            return {
+                "error": f"API request failed: {e}",
+                "text": text,
+                "deployment_type": "serverless"
+            }
+
+    def _predict_endpoint(self, text: str) -> Dict[str, Any]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)